resources/lang/en/events/death/general.json

```diff
@@ -4995,8 +4995,8 @@
                 "cats": [
                     "m_c"
                 ],
-                "reg_death": "m_c died while wandering the territory.",
-                "lead_death": "died while wandering the territory"
+                "reg_death": "m_c was killed by r_c while wandering the territory.",
```

Comment:
as the original author of that event, i had intended for this to be something like "m_c was led out into the territory and killed by r_c." idk if that's actually suitable as a death log but y'know